From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Thu, 1 Sep 2022 12:05:29 +0200
Subject: [PATCH] Remove WAD / data targets from build

We're crosscompiling but PRBoom+ still tries to invoke the rdatawad
tool, which we're supposed to get from a different native build.

We download the PRBoom WAD separately, so we remove it from the build.
---
 CMakeLists.txt     | 2 --
 src/CMakeLists.txt | 1 -
 2 files changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 55e1573..540c85c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -215,7 +215,6 @@ endif()
 
 if(CMAKE_CROSSCOMPILING)
     set(IMPORT_EXECUTABLES "IMPORTFILE-NOTFOUND" CACHE FILEPATH "Export file from native build")
-    include(${IMPORT_EXECUTABLES})
 else()
     if(NOT CROSS_EXPORTS)
         set(CROSS_EXPORTS "")
@@ -227,7 +226,6 @@ set(PRBOOM_OUTPUT_PATH ${CMAKE_BINARY_DIR})
 set(WAD_DATA prboom-plus.wad)
 set(WAD_DATA_PATH "${PRBOOM_OUTPUT_PATH}/${WAD_DATA}")
 
-add_subdirectory(data)
 add_subdirectory(doc)
 add_subdirectory(src)
 
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 627ef88..ae3599b 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -374,7 +374,6 @@ function(AddGameExecutable TARGET SOURCES)
         target_include_directories(${TARGET} PRIVATE ${ALSA_INCLUDE_DIR})
         target_link_libraries(${TARGET} PRIVATE ${ASOUND_LIBRARY})
     endif()
-    add_dependencies(${TARGET} prboomwad)
 
     if(MSVC)
         set_target_properties(${TARGET} PROPERTIES
